PV260_Project/BusinessLayer/Services/IFileUrlService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using DataLayer.Models;
+
+namespace BusinessLayer.Services
+{
+    public interface IFileUrlService

```

Comment:
What about deleting the `FileUrl` from db? We don't need to implement it, just asking.

Reply:
Ok that sounds good too, there is no need to add more logic to this small app.